src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -75,12 +75,17 @@ private TranscriptionEntity saveTranscription(UserAccountEntity userAccount,
         }
 
         TranscriptionEntity transcription = new TranscriptionEntity();
-
+        HearingEntity hearing = null;
         if (!isNull(transcriptionRequestDetails.getHearingId())) {
-            transcription.setHearing(hearingsService.getHearingById(transcriptionRequestDetails.getHearingId()));
+            hearing = hearingsService.getHearingById(transcriptionRequestDetails.getHearingId());
```

Comment:
Really, we just need to getReferenceById to set the FK link - no need to get the full object.
But understand why we're doing that for convenience with the added API errors we have added in those services.